s3-s3-replication-sam/example-pattern.json

```diff
@@ -0,0 +1,60 @@
+{
+  "title": "AWS S3 bucket replication",
+  "description": "This pattern sets up replication between S3 buckets.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "Data Redundancy and Durability: Safeguard against data loss with automatic and asynchronous replication of objects between S3 buckets.",      "Global Content Distribution: Reduce latency and enhance user experience by replicating frequently accessed data to various geographical locations.",
+      "Compliance and Data Retention: Address regulatory standards and compliance requirements by replicating data to separate regions.",
+      "Operational Efficiency: Asynchronous replication allows you to focus on other tasks while data is efficiently copied in the background.",
+      "Same-Region Replication (SRR): Ensure redundancy, compliance, or operational efficiency by copying data to a separate bucket within the same region."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/s3-s3-replication-sam",
+      "templateURL": "serverless-patterns/s3-s3-replication-sam",
+      "projectFolder": "s3-s3-replication-sam",
+      "templateFile": "template.yaml"
+    }
+  },
+  "resources": {
+    "bullets": [
+      {
+        "text": "Replicating Objects",
+        "link": "https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html"
+      },
+      {
+        "text": "Cross Region Replication",
+        "link": "https://www.youtube.com/watch?v=_6admeBN-lI"
```

Comment:
This pattern only sets up replication within the same region. Please include a more relevant link

Reply:
yes, you right. solved